src/components/Promise/Radar.js

```diff
@@ -53,8 +54,8 @@ function Radar(props) {
           zoom={4}
           tolerance={0.001}
           tileLayer={tileLayer}
-          latLng={[-1.28333, 36.81667]}
-          center={[8.7832, 34.5085]}
+          latLng={[lat || -1.28333, long || 36.81667]}
+          center={[lat || 9.082, long || 8.6753]}
```

Comment:
Default values should remain Nairobi... or may be move them to env vars or gSheet itself but Nairobi should remain the default location if nothing has been set.